includes/DetermineApplnPath.php

```diff
@@ -66,6 +66,7 @@
 		{
 			$flag = 1;
 			$postStr = $row2[0];
+			break;
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->